root/components/ReleaseLanguageScript.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+const ReleaseLanguageScript = ({release}: {release: ReleaseT}) => {
```

Comment:
There's another place we can use this in root/report/components/ReleaseList.js, I think, though instead of `null` it uses `lp('-', 'missing data')`. Perhaps that's something we can do everywhere though?

Reply:
Sounds like a good plan, made that change!